cmd/blockchaincmd/deploy.go

```diff
@@ -575,7 +576,15 @@ func deployBlockchain(cmd *cobra.Command, args []string) error {
 					ChangeOwnerAddr:      changeAddr,
 				})
 			}
-		} else {
+
+		case globalNetworkFlags.ClusterName != "":
```

Comment:
are we sure here this is not local cluster?

Reply:
yes we are sure as above this we have a code 
```
if len(bootstrapEndpoints) == 0 {
					bootstrapEndpoints, err = getLocalBootstrapEndpoints()
```
so bootstrapEndoints will be already set and slice len will be >0  for local case

Reply:
added comment for this just in case